tests/grain_size_yield_phases.prm

```diff
@@ -0,0 +1,159 @@
+# This test is like grain_size_phase_function.prm, but
+# it additionally enables plastic yielding in the material model.
```

Comment:
Would it make sense to include the other test? If the only thing you changed is the yielding?